### PR TITLE
docs: correct getting started version

### PR DESCRIPTION
**Pull request by kassianah**
https://app.shortcut.com/mineiros/story/5060/correct-getting-started-example-version

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -78,7 +78,7 @@ section {
       ```hcl
       module "terraform-aws-cognito-user-pool" {
         source  = "mineiros-io/cognito-user-pool/aws"
-        version = "~> 0.9.0"
+        version = "~> 0.9.2"
```
Comment (soerenmartius) [inline README.tfdoc.hcl:81]: Since we're using semantic versioning here, we can leave that value to `~> 0.9.0` which effectively means `0.9.x`

**Review by soerenmartius — CHANGES_REQUESTED** [review]
can be closed

File: README.tfdoc.hcl (hunk 1 above)
Comment (kassianah) [reply README.tfdoc.hcl:81]: Ah I see, that makes sense

**Comment by kassianah** [comment]
Doesn't need to be changed.

**Pull request closed without merging**